frontends/web/src/components/bitboxbase/unlockbitboxbase.tsx

```diff
@@ -0,0 +1,123 @@
+ * limitations under the License.
+ */
+
+import { Component, h, RenderableProps } from 'preact';
+import { alertUser } from '../../components/alert/Alert';
+import { Button } from '../../components/forms';
+import { AppLogo, SwissMadeOpenSource } from '../../components/icon/logo';
+import { Footer, Header } from '../../components/layout';
+import { PasswordSingleInput } from '../../components/password';
+import { translate, TranslateProps } from '../../decorators/translate';
+import { apiPost } from '../../utils/request';
+
+interface UnlockBitBoxBaseProps {
+    bitboxBaseID: string | null;
+}
+
+interface State {
+    password?: string;
+    username: string;
+}
+
+type Props = UnlockBitBoxBaseProps & TranslateProps;
+
+class UnlockBitBoxBase extends Component<Props, State> {
+    constructor(props) {
+        super(props);
+        this.state = {
+            password: undefined,
+            username: 'admin',
+        };
+    }
+
+    private apiPrefix = () => {
+        return 'bitboxbases/' + this.props.bitboxBaseID;
+    }
+
+    private handleFormChange = password => {
+        this.setState({ password });
+    }
+
+    private validate = () => {
+        return this.state.password !== '';
+    }
+
+    private handleSubmit = event => {
+        event.preventDefault();
+        if (!this.validate()) {
+            return;
+        }
+        apiPost(this.apiPrefix() + '/user-authenticate', {username: 'admin', password: this.state.password })
+        .then(response => {
+            if (!response.success) {
+                // TODO: Once error codes are implemented on the base, add them with corresponding text to app.json for translation
+                alertUser(response.message);
+            }
+        });
+        this.setState({ password: '' });
+    }
+
+    public render(
+        {
+            t,
+        }: RenderableProps<Props>,
+        {
+            password,
+        }: State,
+    ) {
+        return (
+            <div class="contentWithGuide">
+                <div className="container">
+                    <Header title={<h2>{t('welcome.title')}</h2>} />
+                    <div className="innerContainer scrollableContainer">
+                        <div className="content narrow padded isVerticallyCentered">
+                            <AppLogo />
+                            <div class="box large">
+                                {
+                                    <form onSubmit={this.handleSubmit}>
+                                        <div className="m-top-default">
+                                            <PasswordSingleInput
+                                                autoFocus
+                                                id="password"
+                                                type="password"
+                                                label="Password"
+                                                placeholder={t('bitboxBaseWizard.pairing.unpaired')}
```

Comment:
A bit too long for a placeholder now :D
> A new BitBoxBase has been connected. Please verify that the following code matches what is shown on your BitBoxBase. If the code matches, touch underneath the check mark on your BitBoxBase and then click the button below to continue.

Reply:
Doesn't just quit fit in the input box :) :)

Reply:
I referenced the wrong thing, bad code monkey. Fixed.